webextensions/README.md

```diff
@@ -0,0 +1,36 @@
+# browser-compat-data
+
+This folder contains compatibility data for the [WebExtensions](https://developer.mozilla.org/Add-ons/WebExtensions) APIs.
+
+It's a JSON file, structured as follows:
+
+* one top level JSON object, which has one property for each WebExtensions API. The name of the property is the name of the module (for example, `"i18n"`, or `"downloads"`). The value is an `api` object.
+
+* an `api` object has one property for each API element (each method, property, type, and event). The name of the property is the name of the element. For example, `"getMessage"` or `"create"`. The value is an `api-element` object.
+
+* an `api-element` is an object with two properties, named: `"desktop_browsers"` and `"mobile_browsers"`. Each of these is a `browser-support` object.
+
+* a `browser-support` object contains a property for each browser for which was are capturing support data.
+
+    * For `"desktop_browsers"`, this is:
+        * `"Chrome"`
+        * `"Opera"`
+        * `"Firefox"`
+        * `"Edge"`
+  
+    * For `"mobile_browsers"`, it is:
+        * `"Firefox OS"`
+        * `"Firefox"`
+        * `"Edge"`.
+
+* each of these browser properties is a `support-statement`.
+
+* a `support-statement` object has:
+
+  * one mandatory property `"support"`, which is a string. This can be any of:
+        * "Yes" (it is supported, but we don't know the version that added support)
+        * "No"
```

Comment:
I think we should go with `true` and `false` instead of "Yes" and "No", so this can't be confused with a version string. And maybe `null` instead of unknown?